Dockerfile

```diff
@@ -5,41 +5,26 @@ FROM rocker/r-ver:4.3
 LABEL base_image="rocker/r-ver:4.3"
 LABEL version="1"
 LABEL software="panelGC"
-LABEL software.version="1.0.0"
-LABEL about.summary="an open source tool for quantifying and monitoring GC bias in sequencing panels"
+LABEL software.version="1.0.1"
+LABEL about.summary="An open source tool for quantifying and monitoring GC bias in sequencing panels"
 LABEL about.home="https://github.com/easygsea/panelGC"
 LABEL about.license="SPDX:GPL-3.0"
 LABEL about.tags="GC bias"
 LABEL maintainer="Murathan Goktas <mgoktas@bcgsc.ca>"
 
-RUN apt-get update \
-        && apt-get install -y --no-install-recommends apt-utils \
-        && apt-get install -y --no-install-recommends \
-        libcurl4-openssl-dev \
-        ## Basic deps
-        gcc \
-        gdb \
-        libxml2-dev \
-        libz-dev \
-        liblzma-dev \
-        libbz2-dev \
-        libgit2-dev \
-        ## sys deps from bioc_full
-        pkg-config \
-        fortran77-compiler \
-        byacc \
-        automake \
-        curl \
-        ## bedtools
-        bedtools \
-        && apt-get clean
+# System dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    apt-utils \
+    libcurl4-openssl-dev \
+    gcc gdb libxml2-dev libz-dev liblzma-dev libbz2-dev libgit2-dev \
+    pkg-config fortran77-compiler byacc automake curl bedtools \
+    openjdk-11-jre-headless git unzip && apt-get clean
 
-RUN install2.r \
-        --error \
-        BiocManager \
-        argparser \
-        data.table \
-        readxl \
-        tidyverse
-
-RUN Rscript -e 'requireNamespace("BiocManager"); BiocManager::install("GenomicRanges"); BiocManager::install("rtracklayer");'
+# R packages
+RUN install2.r --error BiocManager argparser tidyverse
+RUN Rscript -e 'requireNamespace("BiocManager"); BiocManager::install("GenomicRanges"); BiocManager::install("rtracklayer")'
+# Install Nextflow
+ENV NEXTFLOW_VERSION=23.04.0
+RUN curl -s https://get.nextflow.io | bash && \
+    mv nextflow /usr/local/bin/ && \
+    chmod 755 /usr/local/bin/nextflow
```

Comment:
Here we are setting **global** read and execute. Is that because Nextflow require it?

Reply:
I got errors when I did `chmod +x`. Read permissions are required for all 3 permission types.